.github/workflows/ci.yml

```
@@ -126,7 +126,7 @@ jobs:
         os:
           - ubuntu-22.04
           - ubuntu-latest-arm-8-cores
-          - macos-13
+          # - macos-13
```

It would be helpful to leave a note about why it's disabled and when it should be re-enabled:

```suggestion
          # Athena doesn't support Intel Mac
          # - macos-13
```